src/librustdoc/html/format.rs

```diff
@@ -1280,6 +1272,22 @@ impl PrintWithSpace for hir::Mutability {
     }
 }
 
+crate fn print_constness_with_space(
+    c: &hir::Constness,
+    s: Option<&ConstStability>,
+) -> &'static str {
+    match (c, s) {
+        // const stable or no stability attribute
+        (
+            hir::Constness::Const,
+            Some(ConstStability { level: StabilityLevel::Stable { .. }, .. }),
+        )
+        | (hir::Constness::Const, None) => "const ",
+        // const unstable or not const
+        (hir::Constness::Const, _) | (hir::Constness::NotConst, _) => "",
```

Comment:
Listing these patterns doesn't actually filter anything; you can make it a wildcard instead and have the same behavior.
```suggestion
        _ => "",
```